src/ci/docker/x86_64-gnu-tools/checktools.sh

```diff
@@ -85,7 +87,8 @@ status_check() {
     check_dispatch $1 beta rls src/tools/rls
     check_dispatch $1 beta rustfmt src/tools/rustfmt
     check_dispatch $1 beta clippy-driver src/tools/clippy
-    # these tools are not required for beta to successfully branch
+    # These tools are not required on the beta/stable branches.
+    # They will still cause failure during the beta cutoff week, see `src/tools/publish_toolstate.py` for that.
     check_dispatch $1 nightly miri src/tools/miri
     check_dispatch $1 nightly embedded-book src/doc/embedded-book
     check_dispatch $1 nightly rustc-guide src/doc/rustc-guide
```

Comment:
@kennytm should we say that the list of "nightly" tools here and of `REGRESSION_OK` tools in `publish_toolstate.py` should be in sync?

(Ideally they could source from the same data but I am not sure what a good way to do that would be.)

Reply:
Yes ideally these should be in sync.